src/inputFormatter/expressionFormatter.ts

```diff
@@ -70,13 +73,21 @@ export function replaceASingleExpressionElementIntoPrologCode(preformattedExpres
 		;
 }
 
+/**
+ * This function just calls all the other function with the abilaty to detect wrong user input.
+ * @param the pre formatted expression
+ * @param expressionPosition where to match issueCodes with the right position
+ */
 export function expressionIssueDetector(preFormattedExpression: string[], expressionPosition: number) {
 	detectBracketIssues(preFormattedExpression, expressionPosition);
 	detectMissingStatementsOrConnector(preFormattedExpression, expressionPosition);
 }
 
 /**
- * Check the expression for an equal amount of opened and closed brackets.; it can detect wrong input
+ * Check the expression for an equal amount of opened find closed brackets.
```

Comment:
This description suggests that `)foo(` would also count as valid because it's the same number of open and closed brackets. I'd just say something like "Check if each opening bracket is closed correctly and vice versa"